### PR TITLE
Add Aidoo Zigbee from Airzone

### PR DESCRIPTION
**Pull request by AlexisPolegato**
Add Airzone file to integrate Aidoo Zigbee device

### CODE REVIEW

File: src/devices/airzone.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import * as reporting from '../lib/reporting';
+import * as ota from '../lib/ota';
+const e = exposes.presets;
+import tz from '../converters/toZigbee';
+import fz from '../converters/fromZigbee';
+
+const definitions: Definition[] = [
+    {
+        zigbeeModel: ['Aidoo Zigbee'],
+        model: 'Aidoo Zigbee',
```
Comment (Koenkk) [inline src/devices/airzone.ts:12]: Please update with a model number written on the device

File: src/devices/airzone.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import {Definition} from '../lib/types';
```
Comment (Koenkk) [inline src/devices/airzone.ts:1]: Please also add this file to https://github.com/AlexisPolegato/zigbee-herdsman-converters/blob/master/src/devices/index.ts

Comment (Koenkk) [reply src/devices/airzone.ts:1]: @AlexisPolegato can you check this one?

Comment (AlexisPolegato) [reply src/devices/airzone.ts:1]: @Koenkk  I've added here : [3a299a1a0c11186122de53242c1243b92d4b72d7](https://github.com/Koenkk/zigbee-herdsman-converters/pull/7405/commits/3a299a1a0c11186122de53242c1243b92d4b72d7)

**Comment by AlexisPolegato** [comment]
Hello @Koenkk 

Is it possible to approve this PR ?

Thanks,

**Comment by Koenkk** [comment]
I've just merged https://github.com/Koenkk/zigbee-herdsman-converters/pull/7636, could you check if that one also works for you?

**Pull request closed without merging**